helm/nessie/values.yaml

```diff
@@ -0,0 +1,79 @@
+replicaCount: 1
+
+image:
+  repository: projectnessie/nessie
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+# which type of version store to use: JGIT, INMEMORY, DYNAMO. JGIT is best for local testing, DYNAMO preferred for production
+versionStoreType: INMEMORY
+
+# this is needed when selecting DYNAMO
+dynamodb:
+  region: us-west-2
+  awsAccessKeyId: test-key
+  awsSecretAccessKey: test-secret
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations:
```

Comment:
rather than using the scrape annotation I think we should set up service monitors: https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md

see: https://github.com/prometheus-operator/kube-prometheus/pull/16

Reply:
TIL about servicemonitor, will add that

Reply:
done